src/plugins/bfetch/public/streaming/inflate_response.ts

```diff
@@ -7,9 +7,10 @@
  */
 
 import { unzlibSync, strFromU8 } from 'fflate';
+import { toByteArray } from 'base64-js';
```

Comment:
This is an existing dependency and it's a dependency of `Buffer` polyfill. It should weight a lot less then the whole `Buffer`